how-to/how-to-set-prometheus-and-grafana.md

```diff
@@ -0,0 +1,82 @@
+# How to setup Prometheus and `Grafana` as observability platform
+
+Prometheus and `Grafana` are open sources analytics & monitoring solution. This how-to will introduce a simple setup for one Linux node configuration, all Kata Containers, Prometheus and `Grafana` are running on the same host.
+
+> **Warning**: This how-to is only for evaluation purpose, you **SHOULD NOT** running it in production.
+
+* [Install Prometheus](#install-prometheus)
+* [Install `Grafana`](#install-grafana)
+* [Setup `Grafana`](#setup-grafana)
+  * [Create `datasource`](#create-datasource)
+  * [Import dashboard](#import-dashboard)
+
+Both Prometheus and `Grafana` can installed using pre-compiled distributions.
+
+## Install Prometheus
```

Comment:
I wonder if its worth doing this standalone example, versus enabling in Kubernetes. Do you expect folks to be running Prom/Grafana directly on the host, as opposed to daemonset in Kube? Is this just for a simple example?

Reply:
Ah, I see you have the other document. I question if it is worth having both (some duplication of docs), but am okay with keeping this.  What do others think?

Reply:
I would rather have both those in  single document.  I already have the feedback that we need to click on too many links to get to documentation, so I would rather have it as a single document. 
I would also have a link of the document in the Kata 2.0 metrics design doc.